scst-scan/install-app-scanning.hbs.md

```diff
@@ -116,13 +116,23 @@ To install SCST - Scan 2.0:
 
 The following section describes how to configure service accounts and registry credentials. SCST - Scan 2.0 requires the following access:
 
-  - Read access to the registry containing the Tanzu Application Platform bundles. This is the registry from the [Relocate images to a registry](../install-online/profile.hbs.md#relocate-images-to-a-registry) step or `registry.tanzu.vmware.com`.
-  - Read access to the registry containing the image to scan, if scanning a private image
-  - Write access to the registry to which results are published
+| Registry | Permission | Service Account | Example |
```

Comment:
Based on my experience - you are not required to do this if you are just using app-scanning as part of the supply chains right?  I think we should probably put a note at this header that this is optional if you are integrating Scan 2.0 as part of supply chains.  I think the only time you would need to worry about this would be if you are creating standalone IVS templates, right?

Reply:
I guess from reading below - another scenario where you would want to do this is if the registry that the scan image and target image is different than the images that have the tap bundles and target for workload images that are created by TBS.